tools/devops/automation/templates/variables/common.yml

```diff
@@ -64,3 +64,8 @@ variables:
 - name: VSDropsPrefix
   value: 'https://vsdrop.corp.microsoft.com/file/v1/$(BUILD_REPOSITORY_TITLE)/device-tests'
 
+- name: DotNetSdkVersion
+  value: 9.0
+
+- name: DotNetPreviewSdkVersion
+  value: 10.0
```

Comment:
Do we need the .NET 10 preview in the stable .NET 9 branch? If not, I think this should be `9.0` in .NET 9 branch(es) and then bumped to .NET 10 in `net10.0`

Reply:
Nothing explicitly needs it. I kept this at 10.0 so that it can be used in the list of branch triggers in main-stage.yml,
and updated vs-insertion-prep.yml to continue to use the stable version on main instead.